sdk/quantum/quantum-client/README.md

```diff
@@ -0,0 +1,27 @@
+## Azure QuantumClient SDK for JavaScript
+
+This package contains an isomorphic SDK for QuantumClient.
+
+### Currently supported environments
+
+- Node.js version 8.x.x or higher
+- Browser JavaScript
+
+### How to Install
+
+```bash
+npm install @azure/quantum-client
+```
+
+### How to use
+
+#### Sample code
+
+Refer the sample code in the [azure-sdk-for-js-samples](https://github.com/Azure/azure-sdk-for-js-samples) repository.
```

Comment:
this link is broken. Using the most recent code generator to regenerate should fix this problem. For example version: v6.0.0-dev.20210121.1.